src/suite/index.ts

```diff
@@ -402,6 +403,22 @@ export function deployApplicationSuiteToProvider({
     dependsOn.push(migrationJob);
   }
 
+  // Run migration if needed
+  if (migrations) {
+    migrations.map((migration) => {
+      const migrationJob = createMigrationJob(
+        `${name}-migration`,
```

Comment:
you need unique name for it. I'd use the key in the migrations map as name suffix, and value is args

something like 

migrations: {
    "postgres": [args...]
    "ch": [args...]
}